src/translations/en-US.json

```diff
@@ -1,36 +1,38 @@
-	"orc-shared.orcSecretVersion": "Orc-Secret Framework {version}",
-	"orc-shared.orcSharedVersion": "Orc-Shared Framework {version}",
-	"orc-shared.preferences": "Preferences",
-	"orc-shared.save": "Save",
-	"orc-shared.scopeFilterPlaceholder": "Type a scope name",
-	"orc-shared.signOut": "Sign out",
-	"orc-shared.unsavedChanges": "This entity has unsaved changes. All changes will be lost.\n\nAre you sure you want to close it?",
-	"orc-shared.sortOrder": "Sort Order",
-	"orc-shared.status": "Status",
-	"orc-shared.active": "Active",
-	"orc-shared.inactive": "Inactive",
-	"orc-shared.displayName": "Display Name",
-	"orc-shared.description": "Description",
-	"orc-shared.valueName": "Value Name",
-	"orc-shared.delete": "Delete",
-	"orc-shared.remove": "Remove",
-	"orc-shared.showMoreLanguages": "Show more languages",
-	"orc-shared.showFewerLanguages": "Show fewer languages",
+
+  "orc-shared.about": "About",
+  "orc-shared.active": "Active",
+  "orc-shared.allRightsReserved": "All rights reserved.",
+  "orc-shared.cancel": "Cancel",
+  "orc-shared.ccName": "Orckestra Commerce Cloud",
+  "orc-shared.ccVersion": "Commerce Cloud {version}",
+  "orc-shared.close": "Close",
+  "orc-shared.confirmation": "Confirmation",
+  "orc-shared.copyright": "© 2021 Orckestra Technologies Inc.",
+  "orc-shared.copyrightTermsNotice": "This computer program is protected by copyright laws and international treaties. Unauthorized reproduction or redistribution of this program, or any portion of it, may result in severe civil and criminal penalties, and will be prosecuted to the maximum extent possible under the law. Orckestra is a trademark of Orckestra Technologies Inc. All other trademarks are property of the respective owners.",
+  "orc-shared.defaultApp": "Default application",
+  "orc-shared.delete": "Delete",
+  "orc-shared.description": "Description",
+  "orc-shared.displayLanguage": "Display language",
+  "orc-shared.displayName": "Display Name",
+  "orc-shared.fieldIsRequired": "This field is required",
+  "orc-shared.help": "Help",
+  "orc-shared.inactive": "Inactive",
+  "orc-shared.notAvailable": "N/A",
+  "orc-shared.orcScriptsVersion": "Orc-Scripts Framework {version}",
+  "orc-shared.orcSecretVersion": "Orc-Secret Framework {version}",
+  "orc-shared.orcSharedVersion": "Orc-Shared Framework {version}",
+  "orc-shared.preferences": "Preferences",
+  "orc-shared.remove": "Remove",
+  "orc-shared.save": "Save",
+  "orc-shared.scopeFilterPlaceholder": "Type a scope name",
+  "orc-shared.showFewerLanguages": "Show fewer languages",
+  "orc-shared.showMoreLanguages": "Show more languages",
+  "orc-shared.signOut": "Sign out",
+  "orc-shared.sortOrder": "Sort Order",
+  "orc-shared.status": "Status",
+  "orc-shared.unsavedChanges": "This entity has unsaved changes. All changes will be lost.\n\nAre you sure you want to close it?",
+  "orc-shared.valueName": "Value Name",
 	"orc-shared.next": "Next",
```

Comment:
use same indent for all strings

Reply:
It's prettify configs which are defined in orc-scripts. I'm not doing any manual formatting (my ide config is set to use prettify formatting automatically) and if you would open that file you will see the indents are the same for all the file